### DISCUSSION TITLE
Can I use the GLVK keys listed in the docs?

### DISCUSSION THREAD

**Discussion by DmitriiKh** [YerongAI/Office-Tool#697] [Q&A] [answered]
There are GLVK's in this doc file.
doc/Tech Articles/Products.md

Do you know if about any restrictions that wouldn't allow me to use them?

**Comment by YerongAI** [comment, marked as answer]
GVLK is used for KMS activation.

Learn about GVLK: https://learn.microsoft.com/en-us/windows-server/get-started/kms-client-activation-keys

For more info please search on Google.

**Reply by DmitriiKh** [reply]
Thank you.
